devtools/assertions.py

```diff
@@ -51,14 +192,68 @@ def single_none(*args, msg, exception):
 
 
 def single_not_none(*args, msg, exception):
-    
+    r""" Checks in the passed arguments that more than one argument is not
```

Comment:
Please change `more than one argument is not None` to `exactly one argument is not None`

Reply:
Corrected